responsive-grid-layout/onur/global.css

```diff
@@ -8,28 +8,49 @@ body {
   font-family: Arial, sans-serif;
   margin: 0;
 }
+.container {
+    text-align: center;
+    padding: 10px;
+    font-size: 1.5rem;
```

Comment:
If you want to use rem, make sure to first set the font size of root element (usually body) to a certain value. 
See more info here: https://www.freecodecamp.org/news/what-is-rem-in-css/